config.yml

```diff
@@ -323,9 +367,27 @@ jobs:
 
 workflows:
   version: 2
+
+  scheduled_update_check:
+    when: << pipeline.parameters.run-sec-updates-check >>
+    jobs:
+      - static_tests
+      - build
+      - security_updates_check:
```

Comment:
static tests and build really don't need to be here. They will be run when the Pull Request is made.

Reply:
Thank you ! Removed those reqs